src/OrchardCore.Modules/OrchardCore.Users/GraphQL/Startup.cs

```diff
@@ -10,6 +10,6 @@ public class Startup : StartupBase
     public override void ConfigureServices(IServiceCollection services)
     {
         services.AddSingleton<ISchemaBuilder, CurrentUserQuery>();
-        services.AddTransient<UserType>();
+        services.AddSingleton<UserType>();
```

Comment:
Hi @gvkries ， Do you have any other suggestions on how to reuse this type, which would cause an error if you used Transient lifetime?

Could you help to Review this PR? Thanks!

Reply:
The `UserType` is currently only populated by the `CurrentUserQuery` schema builder. It is not individually registered as a type in GraphQL, so I guess that has to change. I need to think about this...